layouts/shortcodes/cs-module.html

```diff
@@ -9,5 +9,5 @@
 
 {{ $name := .Params.name }}
 
-{{ $page := .Site.GetPage (path.Join .Page.Dir "modules" (printf "%s%s" $name ".md"))}}
-{{ $page.Content }}
\ No newline at end of file
+{{ $page := .Site.GetPage (path.Join .Page.File.Dir "modules" (printf "%s%s" $name ".md"))}}
```

Comment:
I'm not 100% sure where the changelog for these changes should go...

Reply:
we don't have a changelog place for this PR's changeset.